.buildkite/gcp/pipeline.yml

```diff
@@ -47,7 +47,7 @@ steps:
           slurm_gpus: 1
           slurm_cpus_per_task: 4
           slurm_ntasks: 1
-          slurm_mem: 30GB
+          slurm_mem: 0GB
```

Comment:
Maybe I don't understand how `slurm_mem` works, but I would have thought this means the job is able to use 0 memory. How is this working now?